### PR TITLE
Introscreen which starts after start is pressed. Shows the storyline of the game. Is skippable.

### PR DESCRIPTION
**Pull request by irenevanderblij**
Introscreen which starts after start is pressed. Shows the storyline of the game. Is skippable.

### CODE REVIEW

File: EscapeApp/app/src/main/java/com/context/kroket/escapeapp/mainscreens/IntroActivity.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import java.util.ArrayList;
+
+/**
+ * Activity responsible for showing the intro (storyline).
+ */
+public class IntroActivity extends AppCompatActivity {
+
+    Handler timerHandler;
+
+    String text0 = "You are an CIA agent.\n\n" +
```
Comment (alanvanrossum) [inline EscapeApp/app/src/main/java/com/context/kroket/escapeapp/mainscreens/IntroActivity.java]: "a" CIA agent. Maybe you can also put here that 'you' are in the headquarters of the cia and not actually present with the man in the room/

**Comment by irenevanderblij** [comment]
Changed that! And changed the list initialization because of a comment that Jochem made in person.

**Comment by alanvanrossum** [comment]
Nice. If someone else confirms this works they can merge it.

**Comment by Faffie** [comment]
Shouldn't this play after START has been received as well?

**Comment by irenevanderblij** [comment]
Woops... guess I forgot that!
Changed it :)

**Comment by Faffie** [comment]
In the blink of an eye! Nicely done! Merging!

**Pull request merged**